modules/telestion-application/src/main/java/de/wuespace/telestion/application/launcher/TelestionConfigParser.java

```diff
@@ -0,0 +1,183 @@
+	 * in the loaders' entry.
+	 */
+	public static final String EXTENDED_LOADER_CONFIG_KEY = "config";
+
+	/**
+	 * The received {@link JsonObject} containing the raw Telestion configuration received upon creation.
+	 */
+	private final JsonObject configuration;
+
+	public TelestionConfigParser(JsonObject configuration) {
+		this.configuration = configuration;
+	}
+
+	/**
+	 * Get the {@link JsonObject} containing the raw Telestion configuration received upon creation.
+	 *
+	 * @return the {@link JsonObject} containing the raw Telestion configuration
+	 */
+	public JsonObject getConfiguration() {
+		return configuration;
+	}
+
+	/**
+	 * <h4>Description</h4>
+	 * Parses the loader configurations on the {@link #LOADERS_KEY loaders JSON property} and creates them.
+	 * <p>
+	 * There are two supported configuration types:
+	 * <ul>
+	 *     <li>The <em>basic configuration</em> is a string which represents the classname of the {@link Loader}.</li>
+	 *     <li>The <em>extended configuration</em> is a {@link JsonObject} which contains the classname and additional
+	 *     configuration options that are passed to the {@link Loader} upon creation.</li>
+	 * </ul>
+	 *
+	 * <h4>Configuration example</h4>
+	 *
+	 * {@code telestion.json}:
+	 * <pre>
+	 * {@code
+	 * {
+	 *   "loaders": [
+	 *     "de.wuespace.telestion.services.loader.DebugLoader",
+	 *     {
+	 *       "className": "de.wuespace.telestion.services.loader.ConfigLoader",
+	 *       "config": {
+	 *         "includeDefaultStores": true
+	 *       }
+	 *     }
+	 *   ]
+	 * }
+	 * }
+	 * </pre>
+	 *
+	 * @return a list of instantiated loaders based on the Telestion configuration
+	 */
+	public List<Loader<?>> parseLoaders()
+			throws IllegalArgumentException, ClassNotFoundException, InvocationTargetException, InstantiationException,
+			IllegalAccessException, NoSuchMethodException {
+		JsonArray loadersConfig;
+		try {
+			loadersConfig = configuration.getJsonArray(LOADERS_KEY);
+		} catch (ClassCastException e) {
+			throw new ClassCastException(("Cannot read loader configuration from property %s. Is the loader " +
+					"configuration a JSON array?").formatted(LOADERS_KEY));
+		}
+		logger.debug("Extracted loaders configuration: {}", loadersConfig);
+
+		List<Loader<?>> loaders = new ArrayList<>();
+		for (Object entry : loadersConfig) {
+			loaders.add(parseLoaderConfig(entry));
+		}
+		logger.debug("Created loaders: {}", loaders);
+
+		return loaders;
+	}
+
+	/**
+	 * Parses the {@link VertxOptions Vert.x options} on the {@link #VERTX_OPTIONS_KEY Vert.x options JSON property}
+	 * and returns them.
+	 *
+	 * @return the parsed {@link VertxOptions Vert.x options}
+	 */
+	public VertxOptions parseVertxOptions() {
+		var vertxOptionsConfig = configuration.getJsonObject(VERTX_OPTIONS_KEY, new JsonObject());
+		logger.debug("Vertx options configuration: {}", vertxOptionsConfig);
+		return new VertxOptions(vertxOptionsConfig);
+	}
+
+	public boolean parseIsClustered() {
```

Comment:
Given that all other `parseXXX` methods have a JavaDoc comment, this one probably should have one, as well 🙂

Reply:
Done in [`0549006` (#504)](https://github.com/wuespace/telestion-core/pull/504/commits/0549006025bd0498fe808db49b0d9674a7830667)